src/Apps/Auction/Routes/__tests__/Register.test.tsx

```diff
@@ -0,0 +1,182 @@
+import React from "react"
+import { graphql } from "react-relay"
+
+import * as Schema from "Artsy/Analytics/Schema"
+import { ErrorModal } from "Components/Modal/ErrorModal"
+import { ModalButton } from "Components/Modal/ModalDialog"
+import { createTestEnv } from "DevTools/createTestEnv"
+import { expectOne } from "DevTools/RootTestPage"
+
+import { RegisterQueryResponseFixture } from "../../__fixtures__/routes_RegisterQuery"
+import { createBidderSuccessful } from "../__fixtures__/MutationResults/createBidder"
+import {
+  createCreditCardFailed,
+  createCreditCardSuccessful,
+} from "../__fixtures__/MutationResults/createCreditCard"
+import { RegisterRouteFragmentContainer } from "../Register"
+import { RegisterTestPage } from "./Utils/RegisterTestPage"
+import { setupCreateTokenMock } from "./Utils/setupCreateTokenMock"
+
+jest.unmock("react-relay")
+jest.unmock("react-tracking")
+jest.mock("Utils/Events", () => ({
+  postEvent: jest.fn(),
+}))
+const mockPostEvent = require("Utils/Events").postEvent as jest.Mock
+
+jest.mock("react-stripe-elements", () => {
+  const stripeMock = {
+    createToken: jest.fn(),
+  }
+
+  return {
+    Elements: ({ children }) => children,
+    StripeProvider: ({ children }) => children,
+    CardElement: ({ onReady, hidePostalCode, ...props }) => <div {...props} />,
+    __stripeMock: stripeMock,
```

Comment:
Looks like this `__stripeMock` accessor is defined and used because of this error if you try pull in any fixtures/functions from other imports within the package mock.

```
babel-plugin-jest-hoist: The module factory of `jest.mock()` is not allowed to reference any out-of-scope variables.
    Invalid variable access: stripeTokenResponse
    Whitelisted objects: Array, ArrayBuffer, Boolean, DataView, Date, Error, EvalError, Float32Array, Float64Array, Function, Generator, GeneratorFunction, Infinity, Int16Array, Int32Array, Int8Array, InternalError, Intl, JSON, Map, Math, NaN, Number, Object, Promise, Proxy, RangeError, ReferenceError, Reflect, RegExp, Set, String, Symbol, SyntaxError, TypeError, URIError, Uint16Array, Uint32Array, Uint8Array, Uint8ClampedArray, WeakMap, WeakSet, arguments, console, expect, isNaN, jest, parseFloat, parseInt, require, undefined, global, process, Buffer, clearImmediate, clearInterval, clearTimeout, setImmediate, setInterval, setTimeout.
    Note: This is a precaution to guard against uninitialized mock variables. If it is ensured that the mock is required lazily, variable names prefixed with `mock` (case insensitive) are permitted.
```